test/gmp.jl

```diff
@@ -445,6 +445,44 @@ end
     @test string(big(0), base = rand(2:62), pad = 0) == ""
 end
 
+@testset "Base.GMP.MPZ.import!/export!" begin
```

Comment:
```suggestion
@testset "Base.GMP.MPZ.export!" begin
```
At this point, the testset is only testing `export!` because `import!` isn't a thing anymore (outside of the testset itself)